model-checks: PaddingConfig codegen emits wrong arity

## Bug

The codegen for Conv padding emits `PaddingConfig2d::Explicit(1, 1, 1, 1)` (4 args) and `PaddingConfig1d::Explicit(1, 1)` (2 args), but the current Burn API expects 2 and 1 arguments respectively.

## Affected models

- **rf-detr** (6 errors, `PaddingConfig2d::Explicit`)
- **yolo** / yolov8n (42 errors, `PaddingConfig2d::Explicit`)
- **silero-vad** (8 errors, `PaddingConfig1d::Explicit`)

## Example error

```
error[E0061]: this enum variant takes 2 arguments but 4 arguments were supplied
   --> .../rf_detr_small.rs:667:27
    |
667 |             .with_padding(PaddingConfig2d::Explicit(1, 1, 1, 1))
    |                           ^^^^^^^^^^^^^^^^^^^^^^^^^       -  - unexpected argument
```

## Reproduction

```sh
cd crates/model-checks/rf-detr
cargo build
# or
cd crates/model-checks/yolo
YOLO_MODEL=yolov8n cargo build
```

Fails on current `main` (commit 6cea4746).

--- a/crates/model-checks/albert/Cargo.toml
+++ b/crates/model-checks/albert/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["tch"]

--- a/crates/model-checks/all-minilm-l6-v2/Cargo.toml
+++ b/crates/model-checks/all-minilm-l6-v2/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["tch"]

--- a/crates/model-checks/clip-vit-b-32-text/Cargo.toml
+++ b/crates/model-checks/clip-vit-b-32-text/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["tch"]

--- a/crates/model-checks/clip-vit-b-32-vision/Cargo.toml
+++ b/crates/model-checks/clip-vit-b-32-vision/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["tch"]

--- a/crates/model-checks/modernbert-base/Cargo.toml
+++ b/crates/model-checks/modernbert-base/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["tch"]

--- a/crates/model-checks/rf-detr/Cargo.toml
+++ b/crates/model-checks/rf-detr/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["ndarray"]

--- a/crates/model-checks/silero-vad/Cargo.toml
+++ b/crates/model-checks/silero-vad/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["ndarray"]

--- a/crates/model-checks/yolo/Cargo.toml
+++ b/crates/model-checks/yolo/Cargo.toml
@@ -7,8 +7,8 @@ publish = false
 [workspace]
 
 [workspace.dependencies]
-burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
-burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4" }
+burn = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
+burn-store = { git = "https://github.com/tracel-ai/burn", default-features = false, rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c" }
 
 [features]
 default = ["tch"]

--- a/examples/raspberry-pi-pico/Cargo.toml
+++ b/examples/raspberry-pi-pico/Cargo.toml
@@ -29,8 +29,8 @@ panic-probe = { version = "1.0.0", features = ["print-defmt"] }
 portable-atomic = { version = "1.7", features = ["critical-section"] }
 embedded-alloc = "0.6.0"
 
-burn = { git = "https://github.com/tracel-ai/burn", rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4", default-features = false, features = ["ndarray"] }
-burn-store = { git = "https://github.com/tracel-ai/burn", rev = "44bbbe8c2cf10784fe3275f9a9c9933732b600e4", default-features = false, features = ["burnpack"] }
+burn = { git = "https://github.com/tracel-ai/burn", rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c", default-features = false, features = ["ndarray"] }
+burn-store = { git = "https://github.com/tracel-ai/burn", rev = "bd991fe52c94faef0cb52dcc92816323a053ad1c", default-features = false, features = ["burnpack"] }
 
 [build-dependencies]
 burn-onnx = { path = "../../crates/burn-onnx" }

